LICENSE

```diff
@@ -13,7 +13,7 @@ The above copyright notice and this permission notice shall be included in all
 copies or substantial portions of the Software.
 
 THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+IMPLIED, INCLUDING BUT NOT LIUNLICENSEDED TO THE WARRANTIES OF MERCHANTABILITY,
```

Comment:
I think this is some kind of copy-paste mistake. Please revert.

Reply:
Fixed